src/main/scala/com/cloudera/sparkts/DateTimeIndex.scala

```diff
@@ -118,27 +124,29 @@ class UniformDateTimeIndex(
     val start: Long,
     val periods: Int,
     val frequency: Frequency,
-    val dateTimeZone: DateTimeZone = DateTimeZone.getDefault())
+    val dateTimeZone: ZoneId = ZoneId.system)
   extends DateTimeIndex {
 
-  override def first: DateTime = new DateTime(start, dateTimeZone)
+  override def first: ZonedDateTime = java.time.ZonedDateTime.ofInstant(java.time.Instant.ofEpochMilli(start), dateTimeZone)
```

Comment:
Currently it does, because the scala-time library doesn't offer the method ofInstant.